BunqSdk/Json/PaginationConverter.cs

```diff
@@ -0,0 +1,129 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Bunq.Sdk.Http;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Bunq.Sdk.Json
+{
+    /// <summary>
+    /// Custom (de)serialization of SessionServer required due to the unconventional structure of the
+    /// SessionServer POST response.
+    /// </summary>
+    public class PaginationConverter : JsonConverter
+    {
+        /// <summary>
+        /// Field constants.
+        /// </summary>
+        private const string FIELD_OLDER_URL = "older_url";
+        private const string FIELD_NEWER_URL = "newer_url";
+        private const string FIELD_FUTURE_URL = "future_url";
+
+        /// <summary>
+        /// Indices of param key and value after parsing.
+        /// </summary>
+        private const int INDEX_PARAM_KEY = 0;
+        private const int INDEX_PARAM_VALUE = 1;
+
+        /// <summary>
+        /// Base dummy URL to hack though the incomplete relative URI functionality of dotnetcore.
```

Comment:
Is though a typo ? Shouldn't it be `through`. And this is kinda clever btw 👏

Reply:
That's indeed a typo! Thanks :)

Reply:
Done